mkdocs.yml

```
@@ -77,6 +77,7 @@ nav:
       - "Force full page OCR": examples/full_page_ocr.py
       - "Accelerator options": examples/run_with_accelerator.py
       - "Simple translation": examples/translate.py      
+      - "Backend converters for XML files": examples/backend_xml_rag.ipynb
```

For Jupyter notebooks the title shown is dictated by the notebook content & cannot be overwritten here, so for clarity I would suggest the following (as done below in RAG section):
```suggestion
      - examples/backend_xml_rag.ipynb
```